src/rpc/misc.cpp

```diff
@@ -66,6 +66,7 @@ UniValue getinfo(const JSONRPCRequest& request)
             "  \"paytxfee\": x.xxxx,         (numeric) the transaction fee set in " + CURRENCY_UNIT + "/kB\n"
             "  \"relayfee\": x.xxxx,         (numeric) minimum relay fee for transactions in " + CURRENCY_UNIT + "/kB\n"
             "  \"errors\": \"...\"           (string) any error messages\n"
+            "  \"deprecation-warning\": \"...\" (string) warning that the getinfo command is deprecated and will be removed in 0.16"
```

Comment:
I guess it's not worth to fix the indentation issue (`"proxy": "host:port",` and `"errors": "..." ` are missing two spaces)?